java/messages/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.nedap.retail.services</groupId>
     <artifactId>messages</artifactId>
-    <version>1.25</version>
+    <version>1.26</version>
```

Comment:
For retail-services-API I want to use the Artifact plugin for Jenkins, so you don't need to update version number here.